eng/build.ps1

```diff
@@ -95,6 +97,7 @@ if ($branding) {
         "default" { $arguments += "" }
     }
 }
+if ($officialBuildId) { $arguments += "/p:OfficialBuildId=$officialBuildId" }
```

Comment:
We should add some validation to this argument. We don't have this validation within the arcade SDK because the use of the property happens before targets execute.